runtimes/pythonrt/local_runner.go

```diff
@@ -126,9 +136,11 @@ func (r *LocalPython) Start(pyExe string, tarData []byte, env map[string]string,
 		"--code-dir", r.userDir,
 	)
 	cmd.Env = overrideEnv(env, r.runnerDir, r.userDir)
+
 	if r.logRunnerCode {
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
+		r.runnerLogger = &zapio.Writer{Log: r.log, Level: zap.InfoLevel}
```

Comment:
In the `remote` project we have a `Log` RPC endpoint. The idea was to add a log handler to the Python logger that will send RPC calls with logs. This way you'll get log levels as well. These logs are meant for the runner only and not the user code. But currently they go to stderr/stdout and will be captured by your logger.

IMO have 2 loggers, one for stdout and one for stderr, each with a `stream` field and also the runtime.